Services/Init/classes/class.ilStartUpGUI.php

```diff
@@ -1263,8 +1271,20 @@ protected function showLogout()
             $this->ctrl->setParameter($this, "client_id", "");
         }
 
+
+        $withdrawal_appendage_text = '';
```

Comment:
Please factor this out to a new `TermsOfService` class. The footprint in `ilStartupGUI` should be as small as possible.